src/core/radio/radio.hpp

```diff
@@ -408,11 +408,24 @@ class Radio : public InstanceLocator, private NonCopyable
     /**
      * This method updates the CSL sample time in radio.
      *
-     * @param[in]  aCslSampleTime  The CSL sample time.
+     * @param[in]  aCslSampleTime   The CSL sample time.
      *
      */
     void UpdateCslSampleTime(uint32_t aCslSampleTime);
 
+    /**
+     * This method schedules a radio reception window at a specific time and duration.
+     *
+     * @param[in]  aChannel   The radio channel on which the frame is to be received.
```

Comment:
nit: we don't know that there will be a frame to receive
```suggestion
     * @param[in]  aChannel   The radio channel on which to receive.
```